### DISCUSSION TITLE
discord server?

### DISCUSSION THREAD

**Discussion by fabse-hack** [cotes2020/jekyll-theme-chirpy#1291] [General]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### What is the topic?

Sharing tips and tricks

### Description

Do we have a Discord server?
Does Jekyll does not have one either :(

i think is a good community thing, you can organizie the different topics and the people can share the knowlage to each other?

**Comment by cotes2020** [comment]
There is no Discord set for Chirpy, you can currently share ideas on the GitHub discussion. See [#ideas](https://github.com/cotes2020/jekyll-theme-chirpy/discussions/categories/ideas).

**Comment by mindlesstux** [comment]
I would add that discord is not indexable.  So when/if I search for a problem if it is jailed to discord or won't be found.

**Reply by fabse-hack** [reply]
in discord, is possible to set an an supporting area .... is indexable, searchable, can open / close the tickets. You can code some own bots on it and make automatic announcements connected to the github repo .....

**Reply by nyhryan** [reply]
I agree, wish I could ask people some stuffs that are not that issue, bugs and get faster replies. I don't know if replying to year old post notifies people haha

**Reply by rafisics** [reply]
Is there any relevant discord server?

**Reply by SoumyaK4** [reply]
@rafisics no

**Reply by rafisics** [reply]
I really wish to get in touch with an active community of this nice theme. This week, I faced multiple issues but didn't get help that much.

**Reply by nyhryan** [reply]
Well you can create an issue and ... wait for a bit 😅

**Reply by rafisics** [reply]
Yes, I am posting them here on discussions and waiting. :)

**Reply by rafisics** [reply]
See, my issues are really simple: https://github.com/cotes2020/jekyll-theme-chirpy/discussions/2257#discussioncomment-12262336

**Reply by rafisics** [reply]
But with this one (https://github.com/cotes2020/jekyll-theme-chirpy/discussions/2258), I don't have any idea what's causing it!
